packages/bitcore-lib/lib/opcode.js

```diff
@@ -75,6 +75,36 @@ Opcode.smallInt = function(n) {
   return new Opcode(Opcode.map.OP_1 + n - 1);
 };
 
+/**
+ * Converts OP_0 - OP_16 to an int
+ * @param {Number} opcode Opcode
+ * @returns {Number}
+ */
+Opcode.decodeOpN = function(opcode) {
+  if (opcode === Opcode.OP_0) {
+    return 0;
+  }
+  $.checkArgument(opcode >= Opcode.OP_1 && opcode <= Opcode.OP_16, new Error('Invalid opcode: ' + JSON.stringify(opcode)));
```

Comment:
check usage (doesn't look like it's supposed to take an Error)

Reply:
It's ok. I added some test cases for these 2 new methods, one of which tests for this.